### ISSUE TITLE
Why cmi5?

### ISSUE THREAD

**Issue by garemoko** [AICC/CMI-5_Spec_Current#380]
There's a question that's been bugging me for a little while so I wanted to ask this group and get a good answer before somebody asks me. The question is this:

<blockquote>
Given that all authoring tools and LMSs that support xAPI follow the same method of packaging and launch that's been stable for a couple of years now, why should a vendor spend dollars moving to, or additionally supporting, cmi5 packaging and launch? 
</blockquote>

All of the why cmi5 blogs and things I've seen have been about moving from SCORM to cmi5, which is fine, but I don't think that's the biggest battle for cmi5 adoption. 

I know part of the answer is that cmi5 builds on the existing launch spec, which makes it a little less work. Hopefully that also means some backwards compatibility? That's not a good enough reason on it's own though. What else am I missing? What's cmi5's killer feature?

**Comment by brianjmiller** [comment]
Just off the top of my head:

1. Defined statement set for session recording
1. Multiple launch handling through session abandonment, etc.
1. Multiple launchable elements (AUs) in a single package (course structure)
1. It is a specification on a potential standards path, the guidelines are not
1. Intentional data points that LMSs are already used to (pass/fail, complete/incomplete, duration, score)
1. Multiple identified launch modes

**Comment by brianjmiller** [comment]
The compatibility question is an interesting one, you could *almost* launch an existing `tincan.xml` package as an AU and have it work, you would only get hung up on the authentication piece, but then that same content is unlikely to be using the "cmi5 defined" statements, etc. and would still need to be converted to an AU as such. (Shameless plug: our SCORM Driver product can already do that.)

**Comment by garemoko** [comment]
Thanks @brianjmiller this is really helpful. I did have some follow up questions on some of those points but i think I answered them by looking at the spec. 

I was also going to ask if any thought had been given to backwards compatibility, but I think that'd just come down to the LMS accepting both types of package and figuring out from the presence of tincan.xml or cmi5.xml which auth process(es) etc. the package supported.
